blueprints/ember-bootstrap/index.js

```diff
@@ -202,5 +203,15 @@ module.exports = {
       let settingsString = JSON.stringify(settings);
       this.ui.writeLine(chalk.red(`Configuration file could not be edited. Manually update your ember-cli-build.js to include '${this.name}': ${settingsString}`));
     }
+  },
+
+  retrieveBootstrapVersion() {
+    let file = 'ember-cli-build.js';
+
+    let source = fs.readFileSync(file);
+    let build = new BuildConfigEditor(source);
+    let config = build.retrieve(this.name);
+
+    return config && config.bootstrapVersion;
```

Comment:
As we are checking for strict equality of a number, so maybe to increase fault tolerancy this should be `config && parseInt(config.bootstrapVersion)`?